src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
```

Comment:
I don't think there is technical drift. We already have a function (`run_command`) that can be used here. Just need to update our tests so when it captures what commands are run the branch is optional.